addons/sys_intercom/stringtable.xml

```diff
@@ -344,44 +344,16 @@
             <French>Activation de la voix</French>
         </Key>
         <Key ID="STR_ACRE_sys_intercom_intercomPttKey">
-            <English>Intercom Key</English>
-            <Spanish>Tecla de intercom</Spanish>
-            <Polish>Klawisz interkomu</Polish>
-            <Italian>Pulsante intercom</Italian>
-            <Japanese>インターコム キー</Japanese>
-            <Chinese>無線電按鍵</Chinese>
-            <Chinesesimp>无线电按键</Chinesesimp>
-            <French>Clé de l'interphone</French>
+            <English>Intercom PTT</English>
         </Key>
         <Key ID="STR_ACRE_sys_intercom_intercomPttKey_description">
-            <English>Use the intercom station to talk.</English>
-            <Spanish>Usa la estación de intercom para hablar.</Spanish>
-            <Polish>Mów przez interkom</Polish>
-            <Italian>Usa la stazione intercom per parlare.</Italian>
-            <Japanese>インターコム内で会話する為に使用します。</Japanese>
-            <Chinese>使用通訊機台來講話</Chinese>
-            <Chinesesimp>使用通讯机台来讲话</Chinesesimp>
-            <French>Utiliser le poste d'interphone pour parler.</French>
+            <English>Activate intercom station push-to-talk.</English>
```

Comment:
Is it standard for these to have full-stops? They technically aren't sentences.

Reply:
They are descriptions - and this is a sentence.